run_in_compat_layer_env.sh

```diff
@@ -1,9 +1,14 @@
 #!/bin/bash
+
+base_dir=$(dirname $(realpath $0))
+source ${base_dir}/init/eessi_defaults
+
+BUILD_CONTAINER="docker://ghcr.io/eessi/build-node:debian10"
```

Comment:
This is not used at all, please remove (probably a copy-paste error from `build_container.sh`

Reply:
fixed in 4d5cbce